src/vs/workbench/services/history/browser/history.ts

```diff
@@ -424,6 +435,10 @@ export class HistoryService extends BaseHistoryService implements IHistoryServic
 		}
 	}
 
+	private isEditorEagerlyPreviewing(editor: IBaseEditor): boolean {
```

Comment:
it was the safeguard to not log the tabs being previewed who had `forcePreview` set.